airflow-core/src/airflow/ui/src/i18n/locales/pl/assets.json

```
@@ -0,0 +1,29 @@
+{
+  "consumingDags": "Pobieranie DAG-ów",
+  "createEvent": {
+    "button": "Utwórz zdarzenie",
+    "manual": {
+      "description": "Utwórz ręcznie zdarzenie zasobu",
+      "extra": "Dodatkowe informacje o zdarzeniu zasobu",
+      "label": "Ręcznie"
+    },
+    "materialize": {
+      "description": "Wyzwól DAG zależny od tego zasobu",
```

```suggestion
      "description": "Wykonaj DAG zależny od tego zasobu",
```
I think it's the same discussion again about consistency, and I think we should go with "Wykonanie/Wykonaj" over "Wyzwalanie/Wyzwól"

---

I see that "Uruchomiony" also appears somewhere when talking about "trigger", do we want to change it as well?

---

I reviewed and corrected it and I think the best one is:

* when we talk about "dag run" -> we talk about "wykonanie"
* but when we talk about "triggering dag run" -> we should talk about "uruchomieniu"

That makes sense because "wykonanie" is more "after" and "uruchomienie" is more "before" it turns into "wykonanie".